example/Basic/routeMapper.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
but we're not using it anywhere. I think it introduce more confusion. If you want to show how you can approach it - feel free to describe the approach in the readme